### DISCUSSION TITLE
Should company-yasnippet complete inside strings or comments?

### DISCUSSION THREAD

**Discussion by jcs090218** [company-mode/company-mode#1171] [Q&A] [answered]
This is more a question than an issue. Is it normal that `company-yasnippet` will try to complete under string/comment?

**Comment by dgutov** [comment, marked as answer]
Yes, I think so. Yasnippet works with text in general, not necessarily with code.

IIRC individual snippets can have context requirements: whether one will work at point, or not. If we don't honor such conditions (i.e. by removing such snippets from the list), that would be a good cause for a bug report.

Sorry for the late reply.
